src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -24,9 +24,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!--
       Defines the ability to understand the configuration for the Razor language service provided by
-      the runtime/toolset packages. Introduced in 2.1
+      the runtime/toolset packages. Introduced in 2.1 via a package reference and in 3.0 it's expected
+      to be included by the SDK.
```

Comment:
I think you should look at the other places we do stuff based on TFM - this should be applied for `netcoreapp3.0` and greater without us having to revisit it.

Reply:
Took a look and didn't find anything concerning